sql/7_0_3-to-7_0_4_upgrade.sql

```diff
@@ -160,6 +160,6 @@ ALTER TABLE `product_registration` ADD `last_ask_version` TINYTEXT;
 ALTER TABLE `product_registration` ADD `options` TEXT COMMENT 'JSON array of scope options';
 #EndIf
 
-#IfNotIndex track_events unique_event_label_url
-ALTER TABLE `track_events` DROP INDEX `unique_event_label`, ADD UNIQUE `unique_event_label_url` (`event_label`, `event_url`(255));
+#IfNotIndex track_events unique_event_label_target
+ALTER TABLE `track_events` DROP INDEX `unique_event_label_url`, ADD UNIQUE `unique_event_label_target` (`event_label`, `event_url`(255), `event_target`(255));
```

Comment:
To avoid issues with lingering indexes, would do following:
```
#IfIndex track_events unique_event_label
ALTER TABLE `track_events` DROP INDEX `unique_event_label`;
#EndIf

#IfIndex track_events unique_event_label_url
ALTER TABLE `track_events` DROP INDEX `unique_event_label_url`;
#EndIf

#IfNotIndex track_events unique_event_label_target
ALTER TABLE `track_events` ADD UNIQUE `unique_event_label_target` (`event_label`, `event_url`(255), `event_target`(255));
#EndIf
```